js/src/ui/Form/InputAddress/inputAddress.js

```diff
@@ -78,7 +78,7 @@ class InputAddress extends Component {
     return (
       <div className={ containerClasses.join(' ') }>
         <Input
-          allowCopy={ allowCopy && (disabled ? value : false) }
+          allowCopy={ allowCopy && ((disabled || readOnly) ? value : false) }
```

Comment:
We want it to be `false` if `value` is falsy, because otherwise (if `allowCopy` isn't a boolean) the copy button would show and copy the passed value (could be `''`)